CHANGELOG.rst

```diff
@@ -1,21 +1,24 @@
 Changelog
 =========
 
-Version 0.1
------------
+
+Version 0.0.1
```

Comment:
0.1.0 is usually the default first version, no?

Reply:
we'll get there

Reply:
ok! :D